docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
+
+Description
+-----------
+
+The SimpleCheck gives you basic information about your datasets and detects the most obvious of datashifts.
```

Comment:
Use references here
```suggestion
:ref:`simple` gives you basic information about your datasets and detects the most obvious of datashifts.
```

Reply:
Also, I don't really get why it would make sense to reference the file you are in. But for consistency reasons .. ok

Reply:
Also, I don't really get why it would make sense to reference the file you are in. But for consistency reasons .. ok